Source/Machine.Fakes.FakeItEasy/FakeItEasyMethodCallOccurrence.cs

```diff
@@ -15,17 +15,17 @@ public FakeItEasyMethodCallOccurrence(IVoidArgumentValidationConfiguration confi
 
         public void Times(int numberOfTimesTheMethodShouldHaveBeenCalled)
         {
-            _configuration.MustHaveHappened(Repeated.Exactly.Times(numberOfTimesTheMethodShouldHaveBeenCalled));
+            _configuration.MustHaveHappened(numberOfTimesTheMethodShouldHaveBeenCalled, global::FakeItEasy.Times.Exactly);
```

Comment:
Not wild about the `global::`, though I know why it's needed. Can we do:

```
using HowManyTimes = FakeItEasy.Times;
```

and then `HowManyTimes.Exactly` instead, or something similar?

Reply:
Sure, I can do that. Will amend.

Reply:
`HowManyTimes` doesn't really convey the proper meaning, though... Would you mind if I renamed the `Times` method to `NumberOfTimes` instead?

Reply:
Yeah, I suck at naming things, feel free to amend.

Reply:
> Would you mind if I renamed the `Times` method to `NumberOfTimes` instead?

I did it anyway, as a separate commit so I can roll it back if you don't like it.

Reply:
Ah hang on, I have a feeling this would be a breaking change, so it might be tricky if you rename this method?

Reply:
Yeah confirmed, this is absolutely a breaking change in the API, I'm afraid we can't rename that method. I would go back to using a `using NumberOfTimes = FakeItEasy.Times` instead.

Reply:
> Yeah confirmed, this is absolutely a breaking change in the API, I'm afraid we can't rename that method.

Oops. Sorry, should have seen that. I'll roll it back.